README.md

```diff
@@ -25,7 +25,7 @@ directory for more details.
 
 # Getting started
 
-MahJax-node provides two libraries, `lib/mj-single.js` and `lib/mj-page.js`. Below are two  very minimal examples -- be sure to check out the examples in `./bin/` for more advanced configurations.
+mathjax-node provides two libraries, `lib/mj-single.js` and `lib/mj-page.js`. Below are two  very minimal examples -- be sure to check out the examples in `./bin/` for more advanced configurations.
```

Comment:
One of the reasons I've always preferred mixed-case names is that if the product name starts with a capital letter, you can use it at the beginning of a sentence without it looking wrong. I always regretted that `jsMath` started with lower case letters, since I had to reword things to avoid using it at the beginning of a sentence.  Oh well.